exasol_script_languages_container_ci_setup/templates/slc_code_build.yaml

```diff
@@ -0,0 +1,150 @@
+              StringNotLike:
+                aws:userId:
+                  - !Sub "${CodeBuildRole.RoleId}:*"
+
+  CodeBuildRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+            - Effect: Allow
+              Principal:
+                  Service:
+                    - codebuild.amazonaws.com
+              Action:
+                - sts:AssumeRole
+      Description: !Sub "IAM Role for ${AWS::StackName}"
+      Path: '/'
+      Policies:
+        - PolicyName: root
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action:
+                  - cloudformation:*
+                  - codebuild:*
+                  - logs:*
+                Resource: '*'
+              - Effect: Allow
+                Action:
+                  - secretsmanager:GetSecretValue
+                Resource:
+                  #- arn:aws:secretsmanager:eu-central-1:620087982706:secret:Dockerhub-pFnwBg
+                  - {{ dockerhub_secret_arn }}
+              - Effect: Allow
+                Action:
+                  - s3:PutObject
+                  - s3:GetObject
+                Resource:
+                  - !Sub "arn:aws:s3:::${ArtifactsBucket}/*"
+  BatchBuildRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+            - Effect: Allow
+              Principal:
+                  Service:
+                    - codebuild.amazonaws.com
+              Action:
+                - sts:AssumeRole
+      Description: !Sub "IAM Role for controlling builds on ${AWS::StackName}"
+      Path: '/'
+      Policies:
+        - PolicyName: root
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action:
+                  - codebuild:StartBuild
+                  - codebuild:StopBuild
+                  - codebuild:RetryBuild
+                Resource: '*'
+  {{project}}CodeBuild:
+    Type: AWS::CodeBuild::Project
+    Properties:
+      Description: {{ project }} Container CodeBuild
+      Triggers:
+        BuildType: BUILD_BATCH
+        Webhook: True
+        FilterGroups:
+          -  - Type: EVENT
+               Pattern: PUSH,PULL_REQUEST_MERGED
+             - Type: HEAD_REF
+               Pattern: ^refs/heads/feature/390_evaluate_aws_code_build_for_ci$
+               ExcludeMatchedPattern: false
+      ServiceRole: !GetAtt CodeBuildRole.Arn
+      BuildBatchConfig:
+         BatchReportMode: REPORT_INDIVIDUAL_BUILDS
+         ServiceRole: !GetAtt BatchBuildRole.Arn
+         TimeoutInMins: 240
+      Artifacts:
+        Location: !Ref ArtifactsBucket
+        OverrideArtifactName: true
+        Type: S3
+      Environment:
+        Type: LINUX_CONTAINER
+        PrivilegedMode: true
+        ComputeType: BUILD_GENERAL1_MEDIUM
```

Comment:
For the initial build job, er might be able to use a smaller instance

Reply:
True!